### PR TITLE
LPS-72710 SF

### PR DESCRIPTION
**Pull request by ealonso**
@jbalsas Could you please review 8b8ba39 and sent to Brian if everything is ok?

### CODE REVIEW

**Comment by jbalsas** [comment]
ci:reopen

@ealonso, is there something I need to do with the `version` issue?

**Comment by pavel-savinov** [comment]
ci:reopen

**Comment by pavel-savinov** [comment]
ci:retest

File: modules/apps/web-experience/asset/asset-categories-selector-web/package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+	"dependencies": {
+		"metal": "^2.5.18",
+		"metal-dom": "^2.5.18",
```
Comment (jbalsas) [inline modules/apps/web-experience/asset/asset-categories-selector-web/package.json]: Do we need this dependency?

File: modules/apps/web-experience/asset/asset-categories-selector-web/package.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+		"metal-soy": "^2.4.5",
+		"metal-state": "^2.6.4",
+		"metal-treeview": "^2.1.0"
```
Comment (jbalsas) [inline modules/apps/web-experience/asset/asset-categories-selector-web/package.json]: Do we need this dependency?

File: modules/apps/web-experience/asset/asset-categories-selector-web/src/main/resources/META-INF/resources/js/SelectCategory.es.js [hunk 3]
```diff
@@ -0,0 +1,160 @@
+import Component from 'metal-component';
```
Comment (jbalsas) [inline modules/apps/web-experience/asset/asset-categories-selector-web/src/main/resources/META-INF/resources/js/SelectCategory.es.js]: We're missing the `metal-component` dependency

**Comment by jbalsas** [comment]
Regarding the error I'm seeing, I think it is caused by a newer version of `metal-cli`. Can you use `4.0.1` directly while I investigate?

Thanks!

**Pull request closed without merging**